docs/SPEC.md

```diff
@@ -3550,7 +3550,107 @@ from(bucket: "telegraf/autogen"):
     |> relativeSearchIndex(n: 14)
 ```
 
+#### Kaufman's Efficiency Ratio
+
+Kaufman's Efficiency Ratio indicator is created by taking the absolute value of CMO and dividing by 100 to give the 0 to 1 range.
```

Comment:
Can you expand CMO to its full name?  I think it will make this easier to understand.  Also, maybe add a sentence that tells what KER does, like `Kaufman's Efficiency Ratio provides a way of measuring the volatility of a series.`